analysis/db_schema.py

```diff
@@ -59,6 +61,25 @@ def define_tables(uri, migrate_enabled = False, fake_migrate_all=False):
         Field('reputation', 'double'),
     )
 
+    db.define_table(
+        'revision_block',
+        Field('page_id', 'reference page'),
+        # These fields are used to know in which order to stitch blocks
+        # together, and also, if there are holes between them.
+        # Id of the last revision before the block
+        Field('prev_revision_id', 'integer'), # long??
+        Field('next_revision_id', 'integer'),
+        # To put blocks in chronological order by page.
+        Field('initial_timestamp', 'datetime'),
```

Comment:
what would be better? Type timestamp or datetime? Considering revisions will be international, would UTC  help us?